subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -228,19 +229,19 @@ uint8_t ll_cig_parameters_commit(uint8_t cig_id, uint16_t *handles)
 		 * handle the throughput. For unframed these must be divisible, if they're not,
 		 * framed mode must be forced.
 		 */
-		cig->iso_interval = cig->c_sdu_interval / ISO_INT_UNIT_US;
+		iso_interval_us = cig->c_sdu_interval;
 
-		if (cig->iso_interval < BT_HCI_ISO_INTERVAL_MIN) {
+		if (iso_interval_us < 5000U) {
 			/* ISO_Interval is below minimum (5 ms) */
-			cig->iso_interval = BT_HCI_ISO_INTERVAL_MIN;
+			iso_interval_us = 5000U;
 		}
 
 #if defined(CONFIG_BT_CTLR_CONN_ISO_AVOID_SEGMENTATION)
 		/* Check if this is a HAP usecase which requires higher link bandwidth to ensure
 		 * segmentation is not invoked in ISO-AL.
```

Comment:
Unrelated to this PR, but how can the controller determine which LE Audio profile is being used? :s

Reply:
Good question, @wopu-ot  ?